server/helpers/tokenGenerator.js

```diff
@@ -0,0 +1,4 @@
+const generateToken = () => Math.random().toString(36).substring(2, 15)
+    + Math.random().toString(36).substring(2, 15);
+
+export default generateToken;
```

Comment:
change the name, as we have a generateToken method in utils as well.

Reply:
in utils the method is called getToken, here we have a generateToken method both are different, this method creates a random token value, whereas getToken gets a token from jwt.